storage/ios/Plugin/StoragePlugin.swift

```diff
@@ -0,0 +1,90 @@
+import Foundation
+import Capacitor
+
+@objc(StoragePlugin)
+public class StoragePlugin: CAPPlugin {
+    private var storage = Storage(withConfiguration: StorageConfiguration())
+
+    @objc func configure(_ call: CAPPluginCall) {
+        let group = call.getString("group")
+        var configuration = StorageConfiguration()
+
+        if group != nil {
+            configuration.group = group!
+        }
```

Comment:
An `if let` would be cleaner and avoid the force unwrap
```suggestion
        if let group = group {
            configuration.group = group
        }
```

Reply:
Thanks, I was wondering why it was upset at me.